test/python/test_stellar.py

```diff
@@ -176,15 +173,15 @@ def test_stellar_swap_valid_2(backend, navigator, test_name):
     performer.perform_valid_swap(backend, navigator, test_name, fees=fees, memo=memo, destination=destination, send_amount=send_amount)
     performer.perform_stellar_tx(backend, fees=fees, memo=memo, destination=destination, send_amount=send_amount)
 
-
 # Make a valid swap and then ask a second signature
 def test_stellar_swap_refuse_double_sign(backend, navigator, test_name):
     performer = StellarValidSwapPerformer()
     performer.perform_valid_swap(backend, navigator, test_name)
     performer.perform_stellar_tx(backend)
 
-    with pytest.raises((ChunkedEncodingError, ConnectionError, ProtocolError, IncompleteRead)):
-        performer.perform_stellar_tx(backend)
+    backend.raise_policy = RAISE_NOTHING
+    rapdu = performer.perform_stellar_tx(backend)
+    assert rapdu.status == Errors.INVALID_INSTRUCTION
```

Comment:
I prefer syntax like :
```
with pytest.raises(ApduError) as e:
    performer.perform_stellar_tx(backend)
assert e.value.code == Errors.INVALID_INSTRUCTION
```
Because changing the raise policy might cause trouble if it is not restored back to it's default state.